src/components/tab-bar/tab.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface Tab {
+    /**
+     * Name of the tab.
+     */
+    name: string;
+
+    /**
+     * Name of the icon to use.
+     */
+    icon?: string;
+
+    /**
+     * True if the tab should be selected.
+     */
+    selected?: boolean;
```

Comment:
What do you think about `active` instead of `selected`? It's the term material uses and we should probably try to minimize confusion by having another name for it when they are so similar.
We should also clarify that only one tab at a time can be active